engine/installation/windows/docker-ee.md

```diff
@@ -67,6 +56,16 @@ full list of prerequisites.
     <snip>
     ```
 
+### (optional) Make sure you have all required updates
+
+Some advanced Docker features (like Swarm) requires that Windows is updated to include the fixes in [KB4015217](https://support.microsoft.com/en-us/help/4015217/windows-10-update-kb4015217) (or a later cumulative patch).
```

Comment:
wee typo: "Docker features (like Swarm) require**s**" -- just remove the extraneous `s` and I'll merge.